src/generators/gen.ts

```diff
@@ -34,6 +47,22 @@ export type TrackingPlan = {
     raw: JSONSchema7;
     schema: Schema;
   }[];
+  screenCalls: {
+    raw: JSONSchema7;
+    schema: Schema;
+  }[];
+  pageCalls: {
+    raw: JSONSchema7;
+    schema: Schema;
+  }[];
+  identifyCalls: {
```

Comment:
Aren't we planning to add `alias` API support also?

Reply:
As of now we have not though about adding Alias in TP, need to discuss with product